doc/modules/linear_model.rst

```diff
@@ -1045,32 +1050,94 @@ the "saga" solver is usually faster.
+  multiclass case.
 
-  * :ref:`sphx_glr_auto_examples_linear_model_plot_logistic_path.py`
+For a comparison of some of these solvers, see [9]_.
 
-  * :ref:`sphx_glr_auto_examples_linear_model_plot_logistic_multinomial.py`
+.. topic:: References:
 
-  * :ref:`sphx_glr_auto_examples_linear_model_plot_sparse_logistic_regression_20newsgroups.py`
+  .. [5] Christopher M. Bishop: Pattern Recognition and Machine Learning, Chapter 4.3.4
 
-  * :ref:`sphx_glr_auto_examples_linear_model_plot_sparse_logistic_regression_mnist.py`
+  .. [6] Mark Schmidt, Nicolas Le Roux, and Francis Bach: `Minimizing Finite Sums with the Stochastic Average Gradient. <https://hal.inria.fr/hal-00860051/document>`_
 
-.. _liblinear_differences:
+  .. [7] Aaron Defazio, Francis Bach, Simon Lacoste-Julien:
+      :arxiv:`SAGA: A Fast Incremental Gradient Method With Support for
+      Non-Strongly Convex Composite Objectives. <1407.0202>`
 
-.. topic:: Differences from liblinear:
+  .. [8] https://en.wikipedia.org/wiki/Broyden%E2%80%93Fletcher%E2%80%93Goldfarb%E2%80%93Shanno_algorithm
 
-   There might be a difference in the scores obtained between
-   :class:`LogisticRegression` with ``solver=liblinear``
-   or :class:`LinearSVC` and the external liblinear library directly,
-   when ``fit_intercept=False`` and the fit ``coef_`` (or) the data to
-   be predicted are zeroes. This is because for the sample(s) with
-   ``decision_function`` zero, :class:`LogisticRegression` and :class:`LinearSVC`
-   predict the negative class, while liblinear predicts the positive class.
-   Note that a model with ``fit_intercept=False`` and having many samples with
-   ``decision_function`` zero, is likely to be a underfit, bad model and you are
-   advised to set ``fit_intercept=True`` and increase the intercept_scaling.
+  .. [9] Thomas P. Minka `"A comparison of numerical optimizers for logistic regression"
+          <https://tminka.github.io/papers/logreg/minka-logreg.pdf>`_
+
+  .. [16] :arxiv:`Simon, Noah, J. Friedman and T. Hastie.
+      "A Blockwise Descent Algorithm for Group-penalized Multiresponse and
+      Multinomial Regression." <1311.6529>`
+
+|details-end|
+
+|details-start|
+**Differences from liblinear**
+|details-split|
+
+There might be a difference in the scores obtained between
+:class:`LogisticRegression` with ``solver=liblinear``
+or :class:`LinearSVC` and the external liblinear library directly,
+when ``fit_intercept=False`` and the fit ``coef_`` (or) the data to
+be predicted are zeroes. This is because for the sample(s) with
+``decision_function`` zero, :class:`LogisticRegression` and :class:`LinearSVC`
+predict the negative class, while liblinear predicts the positive class.
+Note that a model with ``fit_intercept=False`` and having many samples with
+``decision_function`` zero, is likely to be a underfit, bad model and you are
+advised to set ``fit_intercept=True`` and increase the intercept_scaling.
```

Comment:
```suggestion
There might be a difference in the scores obtained between
:class:`LogisticRegression` with ``solver=liblinear`` or
:class:`~sklearn.svm.LinearSVC` and the external liblinear library directly,
when ``fit_intercept=False`` and the fit ``coef_`` (or) the data to be predicted
are zeroes. This is because for the sample(s) with ``decision_function`` zero,
:class:`LogisticRegression` and :class:`~sklearn.svm.LinearSVC` predict the
negative class, while liblinear predicts the positive class. Note that a model
with ``fit_intercept=False`` and having many samples with ``decision_function``
zero, is likely to be a underfit, bad model and you are advised to set
``fit_intercept=True`` and increase the intercept_scaling.
```